components/Ink/src/MDCInkView.m

```diff
@@ -193,7 +205,8 @@ - (CGFloat)maxRippleRadius {
 }
 
 - (void)setMaxRippleRadius:(CGFloat)radius {
-  if (self.inkLayer.maxRippleRadius != radius) {
+  if (!MDCCGFloatEqual(self.inkLayer.maxRippleRadius, radius)) {
+    _maxRippleRadius = radius;
```

Comment:
Don't we need to check for inkStyle before we set maxRippleRadius below?
Otherwise what happens if we setInkStyle:Unbounded, then setRadius:12 ?

Reply:
Also what happens if `maxRippleRadius` is set to a number initially, but because it is bounded `maxRippleRadius` is 0 (because default is Bounded). Then someone changes the radius to 0 explicitly and makes the ink Unbounded. The number retained in `_maxRippleRadius` will show the previous number and not 0, because it doesn't pass the `MDCCGFloatEqual` conditional above.

Reply:
Done.